plugins/outputs/prometheus_client/README.md

```diff
@@ -10,6 +10,16 @@ This plugin starts a [Prometheus](https://prometheus.io/) Client, it exposes all
   # Address to listen on
   listen = ":9273"
 
+  # Use TLS
+  tls = true
+  tls_crt = "/etc/ssl/telegraf.crt"
+  tls_key = "/etc/ssl/telegraf.key"
+
+  # Use http basic authentication
+  basic_auth = true
+  username = "Foo"
+  password = "Bar"
```

Comment:
For basic auth lets do it like in #3496, enable if either username or password is set and no enable flag.

Reply:
Done in https://github.com/influxdata/telegraf/pull/3719/commits/bfe91ac83df717b13ff2092e909eb0db5a452faa